docs/plan/2026-04-29-remote-routee-expansion.md

```
@@ -0,0 +1,23 @@
+# RemoteRouterConfig routee 展開実装計画
+
+## 目的
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)